code/LINDA/LINDA_turf_tile.dm

```diff
@@ -207,7 +207,23 @@
 				if(excited_group)
 					last_share_check()
 
-	air.react()
+	if(air)  //How did this even break?
```

Comment:
After talking it over with Fox, this can't be causing the issues I thought it was,

Going to pull that commit.